apps/o2-blocks/src/spoiler/view.scss

```diff
@@ -0,0 +1,34 @@
+.wp-block-a8c-spoiler {
+	padding: 20px;
+	border: 1px solid #000;
+
+	.spoiler {
+		display: none;
+	}
+
+	.show {
+		display: none;
+	}
+
+	.hide:target + .show {
+		display: inline;
+	}
+
+	.hide:target {
+		display: none;
+	}
+
+	.hide:target ~ .spoiler {
```

Comment:
Interesting, this is the main trick of it. One downside of this approach is that it would update the actual url to add the url fragment to make it work. But maybe tradeoff is okay here since we don't need to add any frontend JS.

https://developer.mozilla.org/en-US/docs/Web/CSS/:target

Reply:
thanks. I shamelessly copied this from an "all-CSS spoiler" and I did want to avoid the client-side JS. the random id is there for the link fragment and I figured it was a decent tradeoff. I don't claim to understand the CSS much at all.

now that you mention it this does seem like it will be awkward in contexts like Calypso that have neither client-side JS nor CSS. to be clear: I'm not worried about those contexts for now. seems like eventually it might be better to do something that actually hides the content until it's revealed or until we can click to the fully-rendered view.